easel/test/widget/file_type_icon_test.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:easel_flutter/screens/creator_hub/creator_hub_view_model.dart';
+import 'package:easel_flutter/screens/creator_hub/widgets/nfts_grid_view.dart';
+import 'package:easel_flutter/utils/constants.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get_it/get_it.dart';
+import 'package:provider/provider.dart';
+
+import '../extensions/size_extension.dart';
+import '../mock/mock_repository.dart';
+import '../mocks/mock_constants.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  final repository = MockRepositoryImp();
+  GetIt.I.registerLazySingleton(() => CreatorHubViewModel(repository));
+
+  group("Test NFT File Type Icons", () {
+    testWidgets(
+      "Image Type NFT Icon Test",
+      (widgetTester) async {
+        await widgetTester.testAppForWidgetTesting(
+          Scaffold(
+            body: ChangeNotifierProvider.value(
+              value: GetIt.I.get<CreatorHubViewModel>(),
+              builder: (context, widget) {
+                return NftGridViewItem(
+                  nft: MOCK_NFT,
+                );
+              },
+            ),
+          ),
+        );
+
+        final fileTypeIcon = find.byKey(const Key(kNFTTypeImageIconKey));
+        expect(fileTypeIcon, findsOneWidget);
+        await widgetTester.pump();
```

Comment:
`await widgetTester.pump()` should be before expect statement, there is no need to write it down after `expect`